### ISSUE TITLE
Really cool works. May I use MaCA as my publicated paper test environment?

### ISSUE THREAD

**Issue by HaiyinPiao** [CETC-TFAI/MaCA#7]
And how do I cite this environment in my paper? May I list the authors names in reference?

**Comment by leonfg** [comment]
Of course. BTW the developers‘ names are Gao Fang, Chen SI and Li Mingqiang. If you need a paper to reference, use [this](https://ieeexplore.ieee.org/document/9040781).
